modules/olmv1-about-target-versions.adoc

```diff
@@ -3,106 +3,70 @@
 // * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
 // * operators/olm_v1/arch/olmv1-operator-controller.adoc
 
-:_mod-docs-content-type: CONCEPT
+:_mod-docs-content-type: REFERENCE
 
-[id="olmv1-about-operator-updates_{context}"]
-= About target versions in {olmv1}
+[id="olmv1-about-target-versions_{context}"]
+= Example custom resources (CRs) that specify a target version
 
-In {olmv1-first}, cluster administrators set the target version of an Operator declaratively in the Operator's custom resource (CR).
+In {olmv1-first}, cluster administrators can declaratively set the target version of an Operator or extension in the custom resource (CR).
 
-If you specify a channel in the Operator's CR, {olmv1} installs the latest release from the specified channel. When updates are published to the specified channel, {olmv1} automatically updates to the latest release from the channel.
+You can define a target version by specifying any of the following fields:
+
+* Channel
+* Version number
+* Version range
+
+If you specify a channel in the CR, {olmv1} installs the latest version of the Operator or extension that can be resolved within the specified channel. When updates are published to the specified channel, {olmv1} automatically updates to the latest release that can be resolved from the channel.
 
 .Example CR with a specified channel
 [source,yaml]
 ----
 apiVersion: operators.operatorframework.io/v1alpha1
 kind: Operator
```

Comment:
I believe `Operator` was renamed to `ClusterExtension` since we introduced semver support.

Should we update docs (separate PR is ok for this) to reflect the change?

Reply:
The change to `ClusterExtension` did not make the cut off for 4.15. I think the change went in right after we branched for 4.16. Bryce looked it up to confirm.